src/client/pages/ChangePasswordPage.tsx

```diff
@@ -20,12 +20,14 @@ export const ChangePasswordPage = () => {
   // if the token expires while the user is on the current page
   if (typeof window !== 'undefined' && tokenExpiryTimestamp) {
```

Comment:
it's probably only executing this on the first render `useEffect(..., [])`

Reply:
good spot! I'll wrap these in a `useEffect`